.github/workflows/phpunit-sqlite.yml

```
@@ -27,7 +27,7 @@ jobs:
 
     strategy:
       matrix:
-        php-versions: ['7.4', '8.0', '8.1']
+        php-versions: ['8.1', '8.2', '8.3']
```

```suggestion
        php-versions: ['8.0', '8.1', '8.2', '8.3']
```

https://github.com/nextcloud/server/blob/stable28/lib/versioncheck.php#L29